### ISSUE TITLE
False positive variable used in loop reported as unused

### ISSUE THREAD

**Issue by mpaluchowski** [cflint/CFLint#211]
The following code triggers a `UNUSED_LOCAL_VARIABLE` `Local variable key is not used in function , consider removing it.`:

```
for (var key in values) {
    foo[key] = values[key];
}
```

**Comment by ryaneberly** [comment]
@mpaluchowski , thanks for the example.  

fixed in the dev branch
